tf2_geometry_msgs/CMakeLists.txt

```diff
@@ -28,8 +28,20 @@ ament_auto_find_build_dependencies(REQUIRED ${required_dependencies})
 # )
 
 if(BUILD_TESTING)
+  find_package(ament_lint_auto REQUIRED)
+
+  list(APPEND AMENT_LINT_AUTO_EXCLUDE
+    ament_cmake_cppcheck
+    ament_cmake_uncrustify
+  )
+  ament_lint_auto_find_test_dependencies()
+
+  ament_cppcheck(LANGUAGE "c++")
+  ament_uncrustify(LANGUAGE "C++")
```

Comment:
Just curious, why is this needed? As in, the exclusion and then the separate function call.